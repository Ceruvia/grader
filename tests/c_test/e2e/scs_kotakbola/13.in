217 999 132
H 66
B 10
H 96
B 58
H 57
E
