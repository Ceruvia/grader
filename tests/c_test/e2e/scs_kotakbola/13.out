219 0 68
Tidak
